### DISCUSSION TITLE
Solid cross-section pretension to be mapped from LS-DYNA

### DISCUSSION THREAD

**Discussion by disruptorwin** [OpenRadioss/OpenRadioss#2723] [Ideas]
Hello dears,

We are testing some gearboxes performance in OpenRadioss environment. The original model was in LS-DYNA. Everything goes so smooth until now only except the pre-tension load on solid bolts we applied in LS-DYNA model. Do we have the plan to also add this function mapping in the near future release?

Thank you very much!

**Comment by mislam-bdl** [comment]
Hi disruptorwin
Have you figured out a way to apply pretension from LSDYNA file to OpenRadioss ?
Any suggestions will be greatly appreciated

**Reply by disruptorwin** [reply]
Hello mislam,

Many thanks for the reply. I think we found how to add pretension load on solid part:

Using keyword /PRELOAD and choose a solid cross-section, we can do this. 

But a new issue arises. This keyword has a stop time which means the preload stresses are applied instantaneously which generates some oscillations. In LS-DYNA, they use a curve to define the preload process which is more steady. Is there any way similar way in Open Radioss?

Thanks!
